.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+[What is wrong? What the expected and current behaviours are?]
+
+
+## Code to reproduce
+[Add indicative code example to reproduce the reported case]
+
+
+## Preliminary solution
+[What should be added/changed/removed to match the expected behaviour?]
+
+
+## References
+[Add links to the related articles, packages, reference examples]
+
+
+## Environment
```

Comment:
Turn into a checklist.

## Environment
If reporting a bug, please include the following infomation about your system. Your scikit-image version is skimage.__version__ in a Python shell.
[ ] Operating system and version
[ ] Python version
[ ] Scikit-Image version
[ ] Method by which scikit-image was installed (e.g., from source, via conda/enpkg, or via apt/yum)

## Traceback
[If reporting a bug, please paste the entire traceback from Python here.]